src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/ProjectHotReloadNotificationService.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Microsoft.VisualStudio.Threading;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.HotReload;
+
+/// <summary>
+/// MEF Export that can be used to be notified when a project enters and exits hot reload 
+/// </summary>
+[Export(typeof(IProjectHotReloadNotificationService))]
+internal class ProjectHotReloadNotificationService : IProjectHotReloadNotificationService
+{
+    [ImportingConstructor]
+    public ProjectHotReloadNotificationService(UnconfiguredProject _)
```

Comment:
It would be useful to add a comment explaining why you are importing the `UnconfiguredProject` even through it isn't used elsewhere in the class.